src/AspNetCore/Src/managedexports.cxx

```diff
@@ -355,4 +357,13 @@ http_cancel_io(
     return pHttpContext->CancelIo();
 }
 
+EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
+VOID
+http_reset_connection(
```

Comment:
Did you add this so it's easier to reset connection?  Or will there be additions to this function later on?

Reply:
This is it. Ideally these functions should be very light weight wrappers around the native objects. We could directly expose each of the functions, but as most methods do more than just one line, I've kept them this way.

Reply:
Also this function is used for request aborted.